src/containerapp/azext_containerapp/_params.py

```diff
@@ -147,6 +147,14 @@ def load_arguments(self, _):
     with self.argument_context('containerapp env show') as c:
         c.argument('name', name_type, help='Name of the Container Apps Environment.')
 
+    with self.argument_context('containerapp env storage') as c:
+        c.argument('name', id_part=None)
+        c.argument('storage_name', options_list=["--storage-name", "-s"], help="Name of the storage.")
+        c.argument('access_mode', id_part=None, arg_type=get_enum_type(["ReadWrite", "ReadOnly"]), help="Access mode for the AzureFile storage.")
```

Comment:
> arg_type=get_enum_type(["ReadWrite", "ReadOnly"])

Can the values of these enumerations be obtained from the Python SDK?

Reply:
@zhoxing-ms yes, eventually, once we have the SDK properly generated & post GA. For. //build ACA will not be using SDK. The ACA team is working on the swagger updates , based on reviews & feedback.

Reply:
Got it, thanks~